test/python/expected/sql_validation_framework/expected_create.out

```diff
@@ -186,6 +187,7 @@ Could not find upgrade tests for procedure babel_datatype_precedence_initializer
 Could not find upgrade tests for procedure initialize_babel_extras
 Could not find upgrade tests for procedure initialize_babelfish
 Could not find upgrade tests for procedure remove_babelfish
+Could not find upgrade tests for procedure sys.analyze_babelfish_catalogs
```

Comment:
Add this test to upgrade schedule

Reply:
Do we want to have upgrade test for a procedure which is supposed to be run `sysadmin` only?

Reply:
O I see